op-acceptor/logging/raw_json_sink.go

```diff
@@ -0,0 +1,208 @@
+package logging
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/ethereum-optimism/infra/op-acceptor/types"
+)
+
+// RawJSONSink writes test results in the raw Go JSON test output format
+// that matches what `go test -json` would produce.
+// This is useful for feeding to tools like gotestsum that expect this format.
+type RawJSONSink struct {
+	logger *FileLogger
+}
+
+// GoTestEvent represents an event in the go test JSON output
+// Matches the format described in Go's test2json package
+type GoTestEvent struct {
+	Time    time.Time `json:"Time"`
+	Action  string    `json:"Action"`
+	Package string    `json:"Package"`
+	Test    string    `json:"Test,omitempty"`
+	Output  string    `json:"Output,omitempty"`
+	Elapsed float64   `json:"Elapsed,omitempty"`
+}
+
+// Consume writes a test result to the raw_go_events.log file in Go's test JSON format
+func (s *RawJSONSink) Consume(result *types.TestResult, runID string) error {
+	// Get the raw_go_events.log file path for this runID
+	baseDir, err := s.logger.GetDirectoryForRunID(runID)
+	if err != nil {
+		return err
+	}
+
+	// Create the raw events file path
+	rawEventsFile := filepath.Join(baseDir, "raw_go_events.log")
```

Comment:
this file name string is used in multiple places, worth extracting to a constant?